### PR TITLE
backend skeleton #1

### PR DESCRIPTION
**Pull request by Lys4nder**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+# link it
+add_executable(tests tests/tests.cc)
+target_link_libraries(
+  tests
+  GTest::gtest_main
+)
+
+include(GoogleTest)
+gtest_discover_tests(tests)
```
Comment (CristianIonutIonescu) [inline CMakeLists.txt:136]: I would have this split into a separate cmakefile, under tests folder. Have a look at the scheleton for client

**Review by CristianIonutIonescu — CHANGES_REQUESTED** [review]
generated sources and headers from proto files should not be in include folder but in a subfolder, generated, in build. We don't need to upload the output of proto into git.

File: proto/person.proto [hunk 2]
```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+package person;
```
Comment (enachebogdan) [inline proto/person.proto:3]: you should remove(or replace with something relevant to our app) this as well.

File: src/main.cc [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+#include "../build/proto_generated/person.pb.h"
```
Comment (enachebogdan) [inline src/main.cc:2]: all references to "person"must be removed. Your main will not compile with current code.

**Review by enachebogdan — APPROVED** [review]
I know this is just a demo of protobuf/grpc, but "person" logic and tests are not relevant for our app. You could either replace it with something related to our app or remove it from the skeleton entirely.

**Review by CristianIonutIonescu — APPROVED** [review]

**Pull request merged**